src/scene_file.cpp

```diff
@@ -111,7 +111,9 @@ void Scene_File::Start() {
 
 	index = latest_slot;
 
+	refresh = true;
 	Update();
+	refresh = false;
 	Refresh();
```

Comment:
I do not really understand why we need the double refresh here? Can this not be shuffled around?

Reply:
I also think this looks more like a hack then a solution